### PR TITLE
Documented analytics and cookies

### LINKED ISSUE #927: Document how to configure SiteAnalyse and related
Make this part of the manual.

* [ ] Document Cookie Groups need to be enabled, disable ad blocker

### PR DESCRIPTION
**Pull request by joeribekker**
Fixes #927

### CODE REVIEW

File: docs/configuration/general/analytics.rst [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+Analytics
+=========
+
+By default, Open Forms does not enable any form of external data analytics 
+tools. However, you can enable these tools within Open Forms. Below, we list
+the integration possibilities within Open Forms.
+
+.. warning::
+
+    If you enable external data analytics tools, you **need** to setup a proper
```
Comment (sergei-maertens) [inline docs/configuration/general/analytics.rst:12]:
```suggestion
    If you enable external data analytics tools, you **need** to set up a proper
```

File: docs/configuration/general/analytics.rst [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    :ref:`cookie policy <configuration_general_cookies>` in accordance with
+    the `GDPR`_ and your local privacy authority.
+
+    Also, these data analytics tools will **only work** if the user allowed 
+    these cookies. If you do not setup cookies, these tools will simply not
```
Comment (sergei-maertens) [inline docs/configuration/general/analytics.rst:17]:
```suggestion
    these cookies. If you do not set up cookies, these tools will simply not 
```

File: docs/configuration/general/cookies.rst [hunk 3]
```diff
@@ -0,0 +1,122 @@
+.. _configuration_general_cookies:
+
+Privacy and cookies
+===================
+
+By default, Open Forms only uses (first party) `strictly necessary cookies`_ 
+(although sometimes confusingly called "functionality cookies"). These cookies
```
Comment (sergei-maertens) [inline docs/configuration/general/cookies.rst:7]:
```suggestion
(although sometimes confusingly called "functional cookies"). These cookies 
```
I think that's how it's usually named?

File: docs/configuration/general/cookies.rst [hunk 4]
```diff
@@ -0,0 +1,122 @@
+.. _configuration_general_cookies:
+
+Privacy and cookies
+===================
+
+By default, Open Forms only uses (first party) `strictly necessary cookies`_ 
+(although sometimes confusingly called "functionality cookies"). These cookies 
+are needed to make Open Forms work: The browser should remembers which form
```
Comment (sergei-maertens) [inline docs/configuration/general/cookies.rst:8]:
```suggestion
are needed to make Open Forms work: the browser has to remember which form 
```

File: docs/configuration/general/cookies.rst [hunk 5]
```diff
@@ -0,0 +1,122 @@
+.. _configuration_general_cookies:
+
+Privacy and cookies
+===================
+
+By default, Open Forms only uses (first party) `strictly necessary cookies`_ 
+(although sometimes confusingly called "functionality cookies"). These cookies 
+are needed to make Open Forms work: The browser should remembers which form 
+someone is currently doing and whether you are authenticated in or not.
```
Comment (sergei-maertens) [inline docs/configuration/general/cookies.rst:9]:
```suggestion
someone is currently filling out and whether you are authenticated or not.
```

File: docs/configuration/general/cookies.rst [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+Although this is optional, if you enable any of the settings 
+for :ref:`analytics <configuration_general_analytics>` you will need to 
+configure some cookie settings first.
+
+.. image:: _assets/cookie_notice.png
+
+.. image:: _assets/manage_cookies.png
+
+
+Configure cookies
```
Comment (sergei-maertens) [inline docs/configuration/general/cookies.rst:58]: I think this is missing the step in the global configuration to specify which cookie group is used for analytics.

**Review by sergei-maertens — CHANGES_REQUESTED** [review]

File: docs/configuration/general/cookies.rst (hunk 3 above)
Comment (joeribekker) [reply docs/configuration/general/cookies.rst:7]: I took it from here: https://gdpr.eu/cookies/

I think "functional" has more the meaining of "working" cookies (as apposed to cookies that do not work).

File: docs/configuration/general/cookies.rst (hunk 6 above)
Comment (joeribekker) [reply docs/configuration/general/cookies.rst:58]: I tackled that in the analytics part.

Comment (joeribekker) [reply docs/configuration/general/cookies.rst:58]: Nope, I did not. Will add.

Comment (joeribekker) [reply docs/configuration/general/cookies.rst:58]: Doh, it was tackled below in the section **Configure privacy**

**Review by sergei-maertens — APPROVED** [review]

**Pull request merged**